api/desecapi/views.py

```diff
@@ -581,7 +581,7 @@ def finalize(self):
             PDNSChangeTracker.track(lambda: DomainList.auto_delegate(domain))
             token = models.Token.objects.create(user=action.user, name='dyndns')
             return Response({
-                'detail': 'Success! Here is the password ("auth_token") to configure your router (or any other dynDNS '
+                'detail': 'Success! Here is the password ("token") to configure your router (or any other dynDNS '
                           'client). This password is different from your account password for security reasons.',
```

Comment:
I would like to make the distinction between token and account password more clear. One attempt:

Success! Here is the authentication token to configure your router, or any other DNS client. It must be used as the password for your client. Never store your account password anywhere in plain text, including in configuration files or router devices.

Reply:
changed to:

Success! Here is the secret token required for updating your domain's DNS information. When configuring a router (or other DNS client), place it into the password field of the configuration. Do not confuse the secret token with your account password! Your password is not needed for DNS configuration, and you should not store it anywhere in plain text.